components/eventing-controller/cmd/eventing-controller/main.go

```diff
@@ -90,6 +102,20 @@ func main() {
 		setupLogger.Fatalw("Failed to initialize subscription manager", "backend", v1alpha1.BEBBackendType, "error", err)
 	}
 
+	if opts.EnableNewCRDVersion {
+		setupLogger.Infow("Starting the webhook server")
+
+		if err = (&v1alpha1.Subscription{}).SetupWebhookWithManager(mgr); err != nil {
+			setupLogger.Fatalw("Failed to create webhook", "error", err)
+			os.Exit(1)
```

Comment:
I think you can omit the `os.Exit(1)` since the `Fatalw` call already does the same

Reply:
removed!!!